src/services/_exampleService.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why is this file labeled with an underscore?

Reply:
It makes it easier to remove example code should someone fork from our repo. Exactly where this documentation (and sample code) ends up long term is still under discussion... so this may change anyway.